src/reader.rs

```diff
@@ -795,6 +796,37 @@ where
         Ok(alias)
     }
 
+    fn read_kml_data(&mut self, mut attrs: HashMap<String, String>) -> Result<Data, Error> {
```

Comment:
Is there a reason for making this `read_kml_data` as opposed to `read_data`? `read_data` is a bit generic but would be consistent with the other method names we have here